workflow/rules/datavzrd.smk

```diff
@@ -85,19 +43,22 @@ rule diffexp_datavzrd:
             patterns=["index.html"],
             labels={"model": "{model}"},
         ),
-    params:
-        model=get_model,
     log:
         "logs/datavzrd-report/diffexp.{model}/diffexp.{model}.log",
+    params:
+        extra="",
+        model=get_model,
+        offer_excel=lookup(within=config, dpath="report/offer_excel", default="false"),
```

Comment:
Default should be a boolean.

Reply:
Already done in response to previous comment.